vmngclient/api/templates.py

```diff
@@ -144,7 +151,7 @@ def attach(self, name: str, device: Device) -> bool:
             ]
         }
         endpoint = "/dataservice/template/device/config/attachcli"
-        response = self.session.post(url=endpoint, data=payload).json()
+        response = cast(dict, self.session.post(url=endpoint, json=payload).json())
```

Comment:
Do we need to cast response on dict? `json()` already converts it into Python dict.